fractions/wildfly/security/src/main/java/org/wildfly/swarm/security/SecurityFraction.java

```diff
@@ -55,6 +61,42 @@ public SecurityFraction applyDefaults() {
                                 }})
 
                         )));
+
+        securityDomain(new SecurityDomain("jboss-web-policy")
```

Comment:
I think this should be a customizer in undertow, as to me it doesn't make sense to have a `jboss-web-policy` in place without undertow